site/en/guides/border-router/docker/cleanup-and-basics.md

```diff
@@ -0,0 +1,108 @@
+# Cleanup and Docker Basics
+
+## Stop OTBR Docker
+
+Use `Ctrl+C` in the terminal window running OTBR Docker to stop the process
+gracefully.
+
+If you are running a simulated RCP, also use `Ctrl+C` to stop the processes for
+`socat` and the RCP node itself.
+
+## Restart OTBR Docker
+
+Follow the same complete procedure in [Run OTBR
+Docker](../../../../guides/border-router/docker/run.md) and [Test
```

Comment:
These relative links should move up only 3 directories, not 4.  Anything in the /docker directory should be 3